docs/tfengine/recipes/audit.md

```diff
@@ -0,0 +1,73 @@
+# Audit Recipe
```

Comment:
Add a note at the top to indicate these files are auto-generated

Reply:
Done